### PR TITLE
.NET MAUI support

### PR DESCRIPTION
**Pull request by jgimenez**
Takes updates form @AlonRom as base, adds documentation, packages as NuGet.

### CODE REVIEW

**Comment by jgimenez** [comment]
The changes are significant, therefore I recommend reviewing the checked-out project, instead of diffs.

File: SDK/Android/BugfenderBinding.cs [hunk 1]
```diff
@@ -1,188 +1,182 @@
...
+
+        public Uri SendCrash(string title, string text)
+        {
+            Java.Net.URL url = Com.Bugfender.Sdk.Bugfender.SendCrash(title, text);
+            if (url == null)
+                return null;
+            return new Uri(url.ToString());
+        }
+
+        /* TODO
```
Comment (KenjiTakahashi) [inline SDK/Android/BugfenderBinding.cs:160]: Should this be here?

**Review by KenjiTakahashi — CHANGES_REQUESTED** [review]

File: SDK/Android/BugfenderBinding.cs (hunk 1 above)
Comment (jgimenez) [reply SDK/Android/BugfenderBinding.cs:160]: This TODO hasn't changed (already present in previous version). I don't think it's wrong for it to be here until someone takes the time to address it.

File: README.md [hunk 2]
```diff
@@ -1,148 +1,134 @@
...
+            appKey = "YOUR APP KEY",
+            // apiUri = new Uri("https://api.bugfender.com"),
+            // baseUri = new Uri("https://dashboard.bugfender.com"),
+            printToConsole = true,
+            nativeCrashReporting = true,
+            mauiCrashReporting = true,
+            logUIEvents = true,
+        });
+        bugfender.WriteLine("Logs for this device are here: {0}", bugfender.DeviceUri.ToString());
+        bugfender.Warning("TAG", "This is a warning");
```
[the remark this answers is not in the document]
Comment (jgimenez) [reply README.md:39]: adding comment

File: README.md [hunk 3]
```diff
@@ -1,148 +1,134 @@
...
+{
+    public MainApplication(IntPtr handle, JniHandleOwnership ownerShip) : base(handle, ownerShip)
+    {
+    }
+
+	public override void OnCreate()
+	{
+		base.OnCreate();
+		BugfenderBinding bugfender = BugfenderBinding.Instance;
+        bugfender.Init(this, new BugfenderOptions
```
[the remark this answers is not in the document]
Comment (jgimenez) [reply README.md:68]: yes

File: README.md [hunk 4]
```diff
@@ -1,148 +1,134 @@
...
+ * `bool ForceEnabled { set; }`: forces the SDK to send logs, even if the device is disabled from the dashboard. Use with caution because there is no way to disable logging from the dashboard if you change your mind, prefer to enable devices from the dashboard instead.
+
+Sending logs:
+
+ * Use `WriteLine("hello world")` as you would use the system-provided `Console`, `Trace` or `Debug` objects.
+ * Use `Fatal()`/`Error()`/`Warning()`/`Info()`/`Debug()`/`Trace()` to log events with a certain level. For example `Error("this shouldn't happen")`
+ * You can also specify a tag to easily group logs that belong to the same category, for example `Error("Networking", "Server responded with status 500")`
+ * Use `void Log(int lineNumber, String method, String file, LogLevel logLevel, String tag, String message)` if you want to provide all details for each log.
+
+**Tip:** If your project is currently `Console`/`Trace`/`Debug` to write logs, you can search & replace them to `Bugfender` to get started quickly.Custom device-associated data:
```
[the remark this answers is not in the document]
Comment (jgimenez) [reply README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -1,148 +1,134 @@
...
+```
+* Replace *YOUR APP KEY* with your Bugfender app key.
+
+## Quick reference
+
+Setup:
+
+ * `apiUri` and `baseUri`: are used to point to a specific Bugfender instance, like a specific region or on-prem instance.
+ * `nativeCrashReporting` and `mauiCrashReporting`: enable crash reports on the native (iOS/Android) and .NET MAUI stacks.
+ * `maximumLocalStorageSize`: changes the amount of storage the log cache can take on disk. The size is in bytes and can be up to 50 megabytes.
```
[the remark this answers is not in the document]
Comment (jgimenez) [reply README.md:94]: ok, adding it to the example as a comment, although I don't expect that much people to use it. But yes, it's weird that all other parameters are in the example but this one.

**Comment by jgimenez** [comment]
note some of the files were changed just because of BOM and CRLF changes. I reverted those changes to make the diff smaller.

**Review by KenjiTakahashi — APPROVED** [review]

**Pull request merged**